app/models/censor_rule.rb

```diff
@@ -56,6 +56,18 @@ def is_global?
     info_request_id.nil? && user_id.nil? && public_body_id.nil?
   end
 
+  def expire_requests
+    if info_request
+      info_request.expire
+    elsif user
+      user.expire_requests
+    elsif public_body
+      public_body.expire_requests
+    else # global rule
```

Comment:
Is this the same set of requests that global rules were applied to before this change? It seems like it will miss hidden requests and embargoed requests - is that intended?